File: app/src/main/java/in/woloo/www/home_details/mvp/HomeDetailsView.java
package in.woloo.www.home_details.mvp;

import android.widget.TextView;

import in.woloo.www.home_details.models.LikeResponse;
import in.woloo.www.home_details.models.LikeStatusResponse;
import in.woloo.www.review.models.ReviewListResponse;

public interface HomeDetailsView {
    void getReviewList(ReviewListResponse reviewListResponse);
    void getLike_Unlike(LikeResponse likeResponse, TextView tv_like);
    void likeStatusSuccess(LikeStatusResponse likeStatusResponse);
    void onRedeemSuccess();
}

File: app/src/main/java/in/woloo/www/application_kotlin/interfaces/HomeDetailsView.kt
package `in`.woloo.www.application_kotlin.interfaces

import android.widget.TextView
import `in`.woloo.www.application_kotlin.model.server_response.LikeResponse
import `in`.woloo.www.application_kotlin.model.server_response.LikeStatusResponse
import `in`.woloo.www.application_kotlin.model.server_response.ReviewListResponse

interface HomeDetailsView {
    fun getReviewList(reviewListResponse: ReviewListResponse?)
    fun getLike_Unlike(likeResponse: LikeResponse?, tv_like: TextView?)
    fun likeStatusSuccess(likeStatusResponse: LikeStatusResponse?)
    fun onRedeemSuccess()
}
